Cargo.toml

```diff
@@ -70,11 +70,11 @@ http3 = ["rustls-tls-manual-roots", "h3", "h3-quinn", "quinn", "futures-channel"
 # Don't rely on these whatsoever. They may disappear at anytime.
 
 # Enables common types used for TLS. Useless on its own.
-__tls = []
+__tls = ["dep:rustls-pemfile"]
```

Comment:
Were these changes actually needed?

Reply:
I think so? `rustls-pemfile` is a small parser crate used in `from_pem_bundle`, and that method is exposed regardless of tls-backend. `rustls-pemfile` was previously only included with the `__rustls` feature, so that's why I added it to "both" using `__tls` (and removed it from `__rustls`). Another option could be to duplicate the inclusion in `native-tls`, but I opted for this thinking it's DRYer.

Reply:
Oh OK, I didn't realize it was now also used for native-tls, that's fine!